userguide/tn_initial.rst

```diff
@@ -165,16 +159,14 @@ available within the %brand% graphical administrative interface.
 The screens are listed in the order that they appear within the
 tree, or the left frame of the graphical interface.
 
-.. note:: iXsystems recommends that you contact your iXsystems
-   Support Representative for initial setup and configuration
-   assistance.
+Please :ref:`contact iXsystems Support <Contacting iXsystems>` for
+initial setup and configuration assistance.
 
-Once the system has been configured and you are familiar with the
+When the system has been configured and you are familiar with the
 configuration workflow, the rest of this document can be used as a
-reference guide to the features built into the %brand% Storage
-Array.
+reference guide to the features built into the %brand% Storage Array.
 
-.. note:: It is important to use the |web-ui| or the console setup
+.. warning:: It is important to use the |web-ui| or the console setup
    menu for all non-ZFS configuration changes. %brand% stores settings
```

Comment:
Let's fix this confusing and misguided sentence. What it really means is that some ZFS settings can be changed at the command line if the user knows what they are doing, and what it should really say is "Do not change things from the command line unless directed to by a support person or a proven expert on FreeNAS or TrueNAS."
```suggestion
    menu for all configuration changes. %brand% stores settings
```